src/session/session_compact.c

```diff
@@ -181,7 +181,7 @@ __compact_handle_append(WT_SESSION_IMPL *session, const char *cfg[])
  * __wt_session_compact_check_timeout --
  *     Check if the timeout has been exceeded.
  */
-int
+static int
```

Comment:
If you are adding the `static` keyword, you should remove the `__wt_` prefix from the function name (so it would be just `__session_compact_check_timeout`).

Reply:
Thank you for noticing! Fixed in 08cce6d1cd376e42aed8f88af43d53f42ab90293.